docs/decision_record.md

```diff
@@ -8,6 +8,46 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 [TOC]
 
+## 2023-07-12 Implementations SHOULD provide sorted_name_length_pairs and comparison endpoint
+
+### Decisions
+
+1. Name of the "names-lengths" attribute should be `sorted-name-length-pairs`.
+2. The `sorted-name-length-pairs` is RECOMMENDED.
```

Comment:
If we say `sorted-name-length-pairs` is recommended we be explaining what it is and how it is computed. Unless it is meant to be s different ADR.

Reply:
Ok, I added it:


1. Lump together each name-length pair from the primary collated `names` and `lengths` into an object, like `{"length":123,"name":"chr1"}`.
2. Canonicalize JSON according to the seqcol spec (using RFC-8785).
3. Digest each name-length pair string individually.
4. Sort the digests lexographically.
5. Add as an undigested, uncollated array to the sequence collection.

@sveinugu do you think this is correct?

Reply:
Looks good to me. 

As a decision record I guess it is fine to not explicitly state the benefits of this array, however we should make sure to do this in a clear and understandable way in the specification itself (at least some of the benefits).